.drone/deploy/runner/start.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+default_runner_name="$(hostname)"
+printf "Runner name (%s): " "${default_runner_name}"
+read -r runner_name
+if [ -z "${runner_name}" ]; then
+  runner_name="${default_runner_name}"
+fi
+
+stty -echo
+printf "RPC secret: "
+read -r rpc_secret
+stty echo
+printf "\\n"
+if [ -z "${rpc_secret}" ]; then
+  >&2 echo "The RPC secret cannot be empty"
+  exit 1
+fi
+
+docker run \
```

Comment:
Should probably give a `--name` here since running multiple drone runners on the same machine isn't the best idea (especially given `--restart always`).